### PR TITLE
more configurable options in helm chart

### PR DESCRIPTION
**Pull request by bitnik**
Added more configurable options into helm chart and updated jupyterhub_config.py for hub image accordingly.

### CODE REVIEW

File: images/hub/jupyterhub_config.py [hunk 1]
```diff
@@ -127,6 +133,8 @@ def camelCaseify(s):
     ('extra_resource_guarantees', 'extraResource.guarantees'),
     ('environment', 'extraEnv'),
     ('profile_list', None),
+    ('lifecycle_hooks', None),
```
Comment (consideRatio) [inline images/hub/jupyterhub_config.py]: The lifecycle_hook was already in the list

Comment (consideRatio) [reply images/hub/jupyterhub_config.py]:
```suggestion
```

**Comment by bitnik** [comment]
thanks :)

**Comment by consideRatio** [comment]
:heart: @bitnik thanks!

**Pull request merged**